### PR TITLE
VW PQ: ACC_System & ACC_GRA_Anziege

### PR DESCRIPTION
**Pull request by CineXWorm**
This merge request is about:

- Fix scaling issues
- Remove unused signals
- Add value descriptions

### CODE REVIEW

**Comment by CineXWorm** [comment]
@jyoung8607 I am pretty sure they would like your insight here.

File: vw_golf_mk4.dbc [hunk 1]
```diff
@@ -1074,41 +1074,38 @@ BO_ 1550 Einheiten_1: 2 XXX
  SG_ MFA_v_Einheit_02 : 0|1@1+ (1,0) [0|1] "" XXX
 
 BO_ 872 ACC_System: 8 XXX
- SG_ CHECKSUM : 0|8@1+ (1,0) [0|255] "" XXX
- SG_ COUNTER : 8|4@1+ (1,0) [0|15] "" XXX
+ SG_ ACS_Checksum : 0|8@1+ (1,0) [0|1] "" XXX
+ SG_ ACS_Zaehler : 8|4@1+ (1,0) [0|1] "" XXX
```
Comment (sshane) [inline vw_golf_mk4.dbc:1078]: We use all capital names for these so the parser can recognize which signals are counters/checksums

**Review by jackking444 — APPROVED** [review]

**Comment by jyoung8607** [comment]
To recap my Discord conversation with @jackking444: this PR does identify and fix a couple of signal bugs. They will have a minor impact on OP longitudinal output. Unfortunately this PR also introduces several new bugs, above and beyond just the renaming of the checksum and counter signal. So, this should not be merged as-is. I will file a superseding PR shortly.

**Comment by jyoung8607** [comment]
Following up to myself:

There are only two signals that needed to be fixed in `ACC_System`, fixes submitted in #791. There is only one signal (plus the message name) that needed to be fixed in `ACC_GRA_Anzeige`, fixes submitted in #792.

The rest of the proposed signal changes are invalid. This PR is superseded and can be closed.

**Comment by sshane** [comment]
> The rest of the proposed signal changes are invalid. This PR is superseded and can be closed.

Closing per comment.

**Pull request closed without merging**